docker/Dockerfile

```diff
@@ -32,12 +33,21 @@ RUN apk update && \
       yaml-dev \
       zlib-dev \
       ncurses-dev \
-      bison \
-      autoconf \
-    && echo "gem: --no-ri --no-rdoc" > /etc/gemrc \
-    && bundle install --system $BUNDLER_ARGS \
-    && apk del .ruby-builddeps \
-    && rm -rf /var/cache/apk/*
+    && echo "gem: --no-ri --no-rdoc" > /etc/gemrc
```

Comment:
I found my yesterday logs from inside the image:
```
/usr/src/metasploit-framework $ ls -la /usr/local/bundle/gems/robots-0.10.1/lib/
total 12
drwxr-xr-x    2 root     root          4096 Apr 26 21:46 .
drwxr-xr-x    4 root     root          4096 Apr 26 21:46 ..
-rw-rw----    1 root     root          3174 Apr 26 21:46 robots.rb
```
As you can see robots.rb got weird 660 permissions